doc/install/install-openshift.md

```diff
@@ -313,8 +308,42 @@ Wait for Kuadrant to be ready as follows:
 kubectl wait kuadrant/kuadrant --for="condition=Ready=true" -n kuadrant-system --timeout=300s
 ```
 
+This will setup and configure a number of Kuadrant subcomponents. Some of these can also take additional configuration:
+
+- Authorino (Enforcement Component for AuthPolicy)
+  - Learn More: (Authorino CRD)[https://docs.kuadrant.io/0.11.0/authorino-operator/#the-authorino-custom-resource-definition-crd]
+- Limitador (Enforcement Component for RateLimitPolicy)
+  - Learn More:(Limitador CRD)[https://docs.kuadrant.io/0.11.0/limitador-operator/#features]
```

Comment:
for stable links, might be better to link to:

https://docs.kuadrant.io/latest/authorino-operator/#the-authorino-custom-resource-definition-crd
https://docs.kuadrant.io/latest/limitador-operator/#features